packages/serverless-contracts/src/contracts/apiGateway/__tests__/lambdaHandler.test.ts

```diff
@@ -49,7 +50,7 @@ describe('apiGateway lambda handler', () => {
       path: '/users/{userId}',
       method: 'GET',
       integrationType: 'httpApi',
-      hasAuthorizer: true,
+      authorizerType: 'cognito',
```

Comment:
I think it would be great to add test cases for the other authorizer types as well